instrumentation/src/main/java/io/opentelemetry/android/instrumentation/startup/InitializationListener.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.instrumentation.startup
+
+import io.opentelemetry.api.OpenTelemetry
+
+/**
+ * Provides callbacks to know the sate of the initialization.
+ */
+interface InitializationListener {
+    /**
+     * Called when the RUM initialization starts.
+     */
+    fun onStart()
```

Comment:
We have found it helpful in upstream to have default no-op bodies for some of these, so that implementations only need to override the parts they actually care about. What do you think?